FreeSurfer/FreeSurferPipeline.sh

```diff
@@ -168,7 +168,7 @@ extra_reconall_args=(${extra_reconall_args_manual[@]+"${extra_reconall_args_manu
 
 #parse booleans
 flair=$(opts_StringToBool "$flairString")
-existing_subject=$(opts_StringToBool "$existing_subjectString")
+existing_session=$(opts_StringToBool "$existing_sessionString")
```

Comment:
For reference, we care more about the terminology the user sees than variable names (the users should edit only the Batch scripts, everything else should be run without modifications, so the user will only see variable names that are in the Batch scripts).  Updating the variable names adds the possibility of creating errors in previously-working code, so I recommend not doing that in the future, particularly in non-longitudinal scripts.

Reply:
Ok, fine with that.